src/main/scala/ch/epfl/bluebrain/nexus/admin/organizations/Organizations.scala

```diff
@@ -42,9 +48,31 @@ class Organizations[F[_]](agg: Agg[F], index: OrganizationCache[F])(implicit F:
       case None =>
         val cmd =
           CreateOrganization(UUID.randomUUID, organization.label, organization.description, clock.instant, caller)
-        evalAndUpdateIndex(cmd, organization)
+        evalAndUpdateIndex(cmd, organization) <* setOwnerPermissions(organization.label, caller)
     }
 
+  def setPermissions(orgLabel: String,
+                     permissions: Set[Permission],
+                     acls: AccessControlLists,
+                     subject: Subject): F[Unit] = {
+    val rootPermissions = acls.value.get(/).flatMap(_.value.value.get(subject)).getOrElse(Set.empty)
+    val orgAcl          = acls.value.get(/ + orgLabel).map(_.value.value).getOrElse(Map.empty)
+    val orgPermissions  = orgAcl.getOrElse(subject, Set.empty)
+    val rev             = acls.value.get(/ + orgLabel).map(_.rev)
+
+    if (rootPermissions == permissions || orgPermissions == permissions) F.unit
+    else iamClient.putAcls(/ + orgLabel, AccessControlList(orgAcl + (subject -> permissions)), rev)
+
+  }
+
+  private def setOwnerPermissions(orgLabel: String, subject: Subject): F[Unit] = {
+    for {
+      permissions <- iamClient.permissions
```

Comment:
I'm thinking that maybe we should have the collection of permissions that we want to apply in the config, because there are quite a few that don't make sense...